roles/swift-common/handlers/main.yml

```diff
@@ -1,12 +1,24 @@
 ---
 - name: pip install swifttool
-  pip: name="/opt/stack/{{ swift.swifttool_dir  }}" extra_args='-i https://pypi.python.org/simple/'
+  pip: name="/opt/stack/{{ swift.swifttool_dir  }}" extra_args="-i https://pypi.python.org/simple/"
```

Comment:
Looking at this, this may break in environments without external internet access. We should be using our own mirror for this, not python.org:

```
pip:
  name: "/opt/stack/{{ swift.swifttool_dir }}"
  extra_args: "-i {{ openstack.pypi_mirror | default(omit) }}"
  virtualenv: "{{ swift.venv_dir }}"
```